src/Stratis.Bitcoin/Consensus/ConsensusManager.cs

```diff
@@ -99,7 +52,7 @@ public class ConsensusManager : IConsensusManager, IDisposable
         /// <inheritdoc />
         public ChainedHeader Tip { get; private set; }
 
-        private readonly Dictionary<uint256, List<OnBlockDownloadedCallback>> callbacksByBlocksRequestedHash;
+        private readonly Dictionary<uint256, List<Action<ChainedHeaderBlock>>> callbacksByBlocksRequestedHash;
```

Comment:
I don't think delegate should be replaced with an action since the delegate has a comment on it which improves readability.

Reply:
I disagree here : the comment was 
```    /// <summary>	
    /// A delegate that is used to send callbacks when a bock is downloaded from the of queued requests to downloading blocks.	
    /// </summary>	
    /// <param name="chainedHeaderBlock">The pair of the block and its chained header.</param>
```
The summary is basically explaining what a callback is more than anything, and the name of the dictionary hints at where the callbacks are called. The param simply describes the class ChainedHeaderBlock.
I am happy to comment the Dictionary if we can't agree that the comment is not that valuable

Reply:
If you don't like a comment, improve it, don't delete it

Reply:
@monsieurleberre Im going to have to agree with @noescape00 on this one, can you please revert this change? We can look at this in following PRs. Please try and keep the changes minimal.